.github/workflows/cloud_regression.yml

```diff
@@ -1,5 +1,7 @@
 name: Trigger Cloud Regression E2E Tests
 on:
+  issue_comment:
+    types: [created, edited]
```

Comment:
This will trigger a runner for every new/edited comment in any issue/PR. The actions log will be full of "issue comment" runs. I guess there is no other way to implement it using GHA and that is the recommended way. I wonder why they removed [pull_request_comment](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_comment-use-issue_comment).

Reply:
Unfortunately, yes. It will get triggered for both issues and PRs initially. Due to this check though, **github.event.issue.pull_request**, it will get aborted if comment is on an issue and not on a PR and later on it will get aborted if comment body isn't **/testit**, but yeah, we'll get flooded with such action triggers, which will quickly get aborted.

Reply:
@Ferroin what do you think?

Reply:
Assuming the exact intent is to allow a one-shot trigger (that is, run once when someone explicitly triggers it), my suggestion would be to:

- Run on the `pull_request` event, specifically on the `labeled` activity.
- Scan for a specific label (I recommend the name `run-ci/cloud-integration`, we’re already using the `run-ci/` prefix for labels for things like this).
- Have the workflow remove the label once it’s triggered the cloud integration workflow (this way it becomes a fire-and-forget type thing, no need to remove the label manually).
- Possibly also run on the `created` activity for the `pull_request` event, but still matching on the label (this would slightly speed things up in the case of knowing a run is needed when creating the PR).
- Possibly add some integration back from the integration test workflow itself to comment on the PR on workflow completion to indicate whether it was successful or failed.

Reply:
Yes, will switch to labels for this and try it. I believe it needs to be on pull_request_target and labeled, since we want them for labels added in netdata/netdata repo and not the PR itself. 

However, I feel we're gonna have the same issue, since workflow will be triggered once a label is added on any PR. One trigger for each label. Then filtering will happen and it will get aborted, but the initial issue remains.

> on:
>   pull_request/pull_request_target:
>     types: [ labeled ]
> 
> jobs:
>   build:
>     if: ${{ github.event.label.name == 'test_that_damned_thing' }}

 A runner will be occupied and then condition evaluation will happen. Maybe though the number of triggers is gonna be lower than using the comment way. 

There is always the option to trigger the workflow manually from inside the repo, with no automation at all at a PR level, but I am pretty sure no-one will ever use that 😅

Reply:
@dimko How do you plan to proceed with this PR?